### PR TITLE
Fixing omp-loop

### LINKED ISSUE #1393: Windows compilation error - reference variable in reduction clause
Hi @rubenzorrilla , I think this goes to you

Visual studio shows an error in drag_utilities from FluidDynamicsApplication as it does not accept reference variables in the reduction clause:

image

### PR DESCRIPTION
**Pull request by roigcarlo**
Fixes #1393. It is quite urgent, just noticed that with this all builds will break because of the appveyor check.

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp [hunk 1]
```diff
@@ -90,14 +90,24 @@ namespace Kratos
...
             it_elem->Calculate(DRAG_FORCE, elem_drag, rModelPart.GetProcessInfo());
-            drag_x += elem_drag[0];
-            drag_y += elem_drag[1];
-            drag_z += elem_drag[2];
+            drag_x_red += elem_drag[0];
+            drag_y_red += elem_drag[1];
+            drag_z_red += elem_drag[2];
         }
+        
+        drag_x += drag_x_red;
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp:108]: I think that we can avoid declaring drag_x, drag_y and drag_z by doing
```c
drag_force[0] += drag_x_red;
drag_force[1] += drag_y_red;
drag_force[2] += drag_z_red;
```

**Review by jcotela — APPROVED** [review]

**Review by rubenzorrilla — APPROVED** [review]
Just a minor comment. In any case, thanks for the fix @roigcarlo !

File: applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp (hunk 1 above)
Comment (roigcarlo) [reply applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp:108]: Ok thx I will change it in another PR, :+1:

**Comment by philbucher** [comment]
Can the branch be deleted?

**Comment by roigcarlo** [comment]
Sure, forgot about it sorry

**Pull request merged**